src/App.js

```diff
@@ -51,14 +53,17 @@ class App extends Component {
                key={item.id}            // Key - to help React with performance
 
                // Additional props (event callbacks, etc.) can be passed here
+              //  icon="flower"
+               icon={item.type}
+               parentCB={this.addPoint}
```

Comment:
For clarity, a name for this callback function like `addPointCallback` would be best practice.